framework/src/governance/README.md

```diff
@@ -61,3 +61,136 @@ You can change the default configuration of the AWS Glue Crawlers associated wit
+
+[example cross account](./examples/datazone-msk-environment-authorizer-cross-account.lit.ts)
+
+### Granting MSK Managed VPC connectivity
+
+For easier cross-account Kafka consumption, MSK Provisioned clusters can use the [multi-VPC private connectivity](https://docs.aws.amazon.com/msk/latest/developerguide/aws-access-mult-vpc.html) feature which is a managed solution that simplifies the networking infrastructure for multi-VPC and cross-account connectivity.
+
+By default, the multi-VPC private connectivity permissions are not configured. You can enable it using the construct properties:
+
+[example vpc](./examples/datazone-msk-environment-authorizer-vpc.lit.ts)
+
+[//]: # (governance.datazone-msk-asset-type)
+# DataZoneMskAssetType
+
+DataZone custom asset type for MSK topics.
+
+## Overview
+
+`DataZoneMskAssetType` is a custom asset type implementation for Kafka topics hosted in MSK clusters. MSK clusters can be provisioned or serverless. Topics can be linked to a Glue Schema Registry. 
+The construct is a CDK custom resource that creates the corresponding DataZone Form Types and Asset Type required to store metadata related to MSK Topics. It includes:
+- A MSK Source Reference Form Type containing metadata about the MSK Cluster including the cluster ARN and type. 
+- A Kafka Schema For Type containing metadata about the topic including the topic name, schema version, Glue Schema Registry ARN and Glue Schema ARN.
+
+![DataZone MSK asset type](../../../website/static/img/datazone-msk-asset-type.png)
+
+## Usage
+
+[example default usage](./examples/datazone-msk-asset-type-default.lit.ts)
+
+## Reusing an existing owner project
+
+The `DataZoneMskAssetType` requires a DataZone project to own the custom asset type. By default, it will create a `MskGovernance` project within the domain but you pass an existing project. 
+The construct will make the IAM custom resource Role a member of the projectto be able to create the asset type and the form types.
+
+[example project](./examples/datazone-msk-asset-type-project.lit.ts)
+
+## Reusing a Custom Asset Type Factory
+
+By default, the `DataZoneMskAssetType` creates its own factory resources required to connect to DataZone and create the custom asset type. But it's possible to reuse a Factory across multiple Custom Asset Types to limit the number of custom resource providers and DataZone project membership:
+
+[example factory configuration](./examples/datazone-msk-asset-type-factory.lit.ts)
+
+[//]: # (governance.datazone-gsr-msk-datasource)
+# DataZoneGsrMskDataSource
+
+DataZone Data Source for MSK Topics assets backed by Glue Schema Registry.
+
+## Overview
+
+`DataZoneGsrMskDataSource` is custom data source for DataZone that can create/update/delete MSK topics assets in DataZone based on a Glue Schema Registry definition. The custom data source can be triggered by a schedule or based on events from the Glue Schema Registry. The constructs implement:
```

Comment:
Can we clarify what events we are talking about here? It seems like only Create and Delete. And lets be clear that this will lead to a deletion of an asset in datazone.

Reply:
DataZoneGsrMskDataSource` is custom data source for DataZone that can create/update/delete MSK topics assets in DataZone based on a Glue Schema Registry definition. The custom data source can be triggered by a schedule or based on Create, Update or Registering a new Schema Version events from the Glue Schema Registry. The constructs implement: